scalalib/src/dependency/versions/ModuleDependenciesVersions.scala

```diff
@@ -3,10 +3,10 @@ package mill.scalalib.dependency.versions
 import mill.scalalib.JavaModule
 
 private[dependency] final case class ModuleDependenciesVersions(
-    module: JavaModule,
+    modulePath: String,
     dependencies: Seq[DependencyVersions])
 
 private[dependency] final case class DependencyVersions(
-    dependency: coursier.Dependency,
```

Comment:
Wouldn't be `coursier.Dependency` the better type for later inspection of possible updates?

Reply:
yes indeed. I wasn't sure if coursier should be exposed by mill, so I used a string instead. If it's ok to leak coursier, then I'd be happy to use that instead

Reply:
I guess, mill already leaks `coursier.Dependency` as it's an public property of `mill.scalalib.Dep`. So I think it would be ok expose `coursier.Dependency` here. We currently have no other abstraction / API for managed dependencies.